content/doc/guides/using-common-filters.dmark

```diff
@@ -54,9 +54,9 @@ title: "Using common filters"
 
     #listing[lang=ruby]
       [:default, :sourcemap].each do |rep_name|
-        compile '/css/**/*.sass', rep: rep_name do
+        compile '/css/**/*.scss', rep: rep_name do
           path = @item.identifier.without_ext + '.css'
-          options = { syntax: :sass, style: :compact, sourcemap_path: path + '.map' %}
+          options = { syntax: :sass, style: :compact, css_path: path, sourcemap_path: path + '.map' %}
```

Comment:
The pattern is `'/css/**/*.scss'` but the syntax is `:sass` — I think either the pattern or the syntax needs to be updated.

Reply:
Oops, fixed